src/components/molecules/AVOption/AVOption.vue

```diff
@@ -66,10 +68,19 @@ const ranked = computed(() => props.contest.markingType.voteVariation === "ranke
 
 const checkedCount = computed(
   () =>
-    props.selections.filter((selection) => selection.reference === props.option.reference).length,
+    props.selections.filter(
+      (selection: OptionSelection) => selection.reference === props.option.reference,
+    ).length,
 );
 
-const title = computed(() => props.option.title[i18nLocale.value] ?? "");
+const title = computed(
+  () =>
+    getMeaningfulLabel(
+      props.option as unknown as IterableObject,
```

Comment:
Is double as needed?
```suggestion
      props.option as IterableObject,
```

Reply:
Nope. TS knows that the "props.option" is of type `OptionContent` and that type does not overlap fully with the other. Therefore it is necessary to first cast it as `unknown`